w2w/w2w.py

```diff
@@ -112,11 +114,23 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
         'natural land cover (DEFAULT: 45)',
         default=45,
     )
+
+    parser.add_argument(
+        '-a',
+        '--npix-area',
+        metavar='',
+        type=int,
+        dest='NPIX_AREA',
+        help='Area in number of pixels to look for the nearest number of pixels'
+        'for sampling neighbouring natural land cover (DEFAULT: NPIX_NLC**2)',
+        default=None,
```

Comment:
should this default then not be 45**2?

Reply:
Nope, because if you set NPIX_NLC to something different than 45, then the NPIX_AREA will still be 45**2 by default. It is None here, but it is later set to NPIX_NLC**2 if not defined. I needed the default to be dynamic. It is more common to do this right after parsing the arguments if NPIX_AREA has not been specified by the user, but NPIX_AREA is only used within wrf_remove_urban. We can place it elsewhere if you think it is better.

Reply:
Of course, you are right. Then we can leave this as is!